package/src/components/AutoCompleteInput/AutoCompleteSuggestionHeader.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+import { useTheme } from '../../contexts/themeContext/ThemeContext';
+import { useTranslationContext } from '../../contexts/translationContext/TranslationContext';
+import { Lightning } from '../../icons/Lightning';
+import { Smile } from '../../icons/Smile';
+
+export type AutoCompleteSuggestionHeaderPropsWithContext = {
+  type: string;
+  value?: string;
```

Comment:
The reason is that the `closeSuggestion` util in the AutocompleteInput.tsx is set to `''` when we close a suggestion list so I kept it this way. If we experiment with something on this there's a couple of type errors in the app.

Reply:
I think it definitely should be renamed at the very least, I can help you out with any type errors if you'd like. `value` for example says nothing about what it is or how it's used. 

We can also set it to null rather than an empty string when it's closed for example, which would make more sense, and probably make it more robust to deal with.

Reply:
queryText?

Reply:
queryText makes more sense to me yeah

Reply:
Already renamed value to queryText :)